docs/OV_Runtime_UG/supported_plugins/config_properties.md

```diff
@@ -0,0 +1,226 @@
+
+OpenVINO property is represented as a named constexpr variable with a given string name and type (see ). Example:
+```
+static constexpr Property<std::vector<std::string>, PropertyMutability::RO> available_devices{"AVAILABLE_DEVICES"};
+```
+represents a read-only property with C++ name `ov::available_devices`, string name `AVAILABLE_DEVICES` and type `std::vector<std::string>`.
+
+Refer to the [Hello Query Device С++ Sample](../../../samples/cpp/hello_query_device/README.md) sources and the [Multi-Device execution](../multi_device.md) documentation for examples of using setting and getting properties in user applications.
+
+### Get a set of available devices
+
+Based on read-only property `ov::available_devices`, OpenVINO Core collects information about currently available devices enabled by OpenVINO plugins and returns information using the `ov::Core::get_available_devices` method:
+
+@sphinxdirective
+
+.. tab:: C++
+
+    .. doxygensnippet:: docs/snippets/ov_properties_api.cpp
+       :language: cpp
+       :fragment: [get_available_devices]
+
+.. tab:: Python
+
+    .. doxygensnippet:: docs/snippets/ov_properties_api.py
+       :language: python
+       :fragment: [get_available_devices]
+
+@endsphinxdirective
+
+The function returns a list of available devices, for example:
+
+```
+MYRIAD.1.2-ma2480
+MYRIAD.1.4-ma2480
+CPU
+GPU.0
+GPU.1
+```
+
+If there are more than one instance of a specific device, the devices are enumerated with `.suffix` where `suffix` is a unique string identifier. Each device name can then be passed to:
+
+* `ov::Core::compile_model` to load the model to a specific device with specific configuration properties.
+* `ov::Core::get_property` to get common or device specific properties.
+* All other methods of the `ov::Core` class that accept `deviceName`.
+
+### Working with properties in Your Code
+
+The `ov::Core` class provides the following method to query device information, set or get different device configuration properties:
+
+* `ov::Core::get_property` - Gets the current value of a specific property.
+* `ov::Core::set_property` - Sets a new value for the property globally for specified `device_name`.
+
+The `ov::CompiledModel` class is also extended to support the properties:
+
+* `ov::CompiledModel::get_property`
+* `ov::CompiledModel::set_property`
+
+For documentation about OpenVINO common device-independent properties, refer to `openvino/runtime/properties.hpp`. Device specific configuration keys can be found in corresponding device folders (for example, `openvino/runtime/intel_gpu/properties.hpp`).
+
+### Working with properties via Core
+
+#### Getting device properties
+
+The code below demonstrates how to query `HETERO` device priority of devices which will be used to infer the model:
```

Comment:
`Hetero` execution

Reply:
but in the snippet below HETERO is used as device name